dts/bindings/sensor/vishay,vcnl36825t.yaml

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2023 Juliane Schulze, deveritec GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  VCNL36825T proximity and ambient light sensor.  See datasheet at
+  https://www.vishay.com/docs/84274/vcnl36825t.pdf
+
+compatible: "vishay,vcnl36825t"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  operation-mode:
+    type: string
+    default: "auto"
+    enum:
+      - "auto"
+      - "force"
+    description: |
+      Mode of operation.
+      In "auto" mode the sensor performs sampling the continuously,
+      in "force" mode the sampling is performed on every fetch command.
+
+  measurement-period:
+    type: int
+    default: 40
```

Comment:
The default needs to be justified in the description. Is it a register reset value?

This needs to be addressed in the other properties as well.

Reply:
added the justification. Is this okay?

Reply:
Yes, thanks